CHANGELOG.md

```diff
@@ -28,6 +28,10 @@ Change log itself follows [Keep a CHANGELOG](http://keepachangelog.com) format.
 
 ## 0.11.1
 
+### Added
+
+* `Faker.Address.It`, `Faker.Color.It`, `Faker.Internet.It` and `Faker.Name.It` [[@pmontrasio][]]
+
```

Comment:
These changes should be in the `unreleased` second.

Reply:
Done.